build.gradle.kts

```diff
@@ -10,7 +10,10 @@ repositories {
 }
 
 configurations {
-  testImplementation { exclude(group = "org.junit.vintage") }
+  testImplementation {
+    exclude(group = "org.junit.vintage")
+    exclude("ch.qos.logback")
```

Comment:
Excluded for the tests to prevent conflict with slf4j-test. (Is this an issue?)
Details here: https://projects.lidalia.org.uk/slf4j-test/